### ISSUE TITLE
Dance speed should affect more than cooldown after attacks

### ISSUE THREAD

**Issue by natis1** [natis1/infinitegrimm#6]

**Comment by natis1** [comment]
Please suggest things I could increase the speed of, and the highest speed they should be at, without making the fight physically impossible.

**Comment by natis1** [comment]
I'm moving this to 0.3 because although I have dumped a list of the animations I'm not sure which ones I can safely change without making the fight unbalanced. Here's a list of the ones not currently edited that might be interesting:

```//ground slash?
Slash 3
Slash 2
Slash 1
Slash Antic
// firebats
Cast Antic
Cast
Cast Return
// airdash (where he strikes down and towards you)
Air Dash Antic
Air Dash
G Dash Antic
G Dash
// spikes
Capespike Cast
// misc
Evade Antic
Evade
Evade End
Explode Antic // stun
Reforming // stun

In additon there's several attacks controlled by simple waits, like pillars, part of the recovery after all attacks, and recovery after spikes. For now I think it's fine though so I'm switching this to release 0.3

**Comment by natis1** [comment]
Fixed in 0.2.5 - the hardmode update
